### PR TITLE
Citation report update

### PR DESCRIPTION
**Pull request by florisvdh**
Updates in the section on Better BibTeX (issue solved + a non-issue).

### CODE REVIEW

File: source/citation_style/index.Rmd [hunk 1]
```diff
@@ -118,21 +118,17 @@ Deze worden behandeld in hoofdstuk \@ref(bibcsl).
...
-Hiervan is een [issue](https://github.com/retorquere/zotero-better-bibtex/issues/1601) gemaakt bij Better BibTeX.
-
 ### Export van article met page-range 'x-y'
 
 Range `x-y` wordt omgezet naar `x--y` (in bib-file).
 
-Zie aangemaakt [issue](https://github.com/retorquere/zotero-better-bibtex/issues/1602).
+Zie bespreking in het [issue](https://github.com/retorquere/zotero-better-bibtex/issues/1602).
+Dit blijkt een weloverwogen feature te zijn (en geen bug) om met zekerheid n-dashes te induceren voor page-ranges.
+Zie ook: '[many, but not all, BibTeX styles will correct hyphen to n-dash]( many, but not all, BibTeX styles will correct hyphen to n-dash)'.
```
Comment (ThierryO) [inline source/citation_style/index.Rmd]: I think the URL needs fixing

**Review by ThierryO — CHANGES_REQUESTED** [review]
Looks good, expect for a failing URL.

File: source/citation_style/index.Rmd (hunk 1 above)
Comment (florisvdh) [reply source/citation_style/index.Rmd]: well spotted, will fix it

**Review by ThierryO — APPROVED** [review]

**Comment by florisvdh** [comment]
In attendance of a fix for #28, I upload here the locally rendered files so that they can be compared with the website version after this PR will be merged.

- [citation_style.zip](https://github.com/inbo/inbomd_examples/files/5141942/citation_style.zip) (this is the gitbook)
- [citation_style_bst.pdf](https://github.com/inbo/inbomd_examples/files/5141943/citation_style_bst.pdf)
- [citation_style_csl.pdf](https://github.com/inbo/inbomd_examples/files/5141944/citation_style_csl.pdf)

Done with INBOmd@`6a00c0b`. (current master)

**Pull request merged**